app/src/main/res/layout/featured_search_result_view.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout android:id="@+id/search_term_view"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  android:orientation="vertical"
+  android:paddingEnd="@dimen/feed_padding_x"
+  android:paddingLeft="@dimen/feed_padding_x"
+  android:paddingRight="@dimen/feed_padding_x"
+  android:paddingStart="@dimen/feed_padding_x"
+  xmlns:card_view="http://schemas.android.com/apk/res-auto">
+
+  <android.support.v7.widget.CardView
+    android:layout_marginTop="@dimen/grid_new_2"
+    card_view:cardUseCompatPadding="true"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <RelativeLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:orientation="horizontal">
+
+      <ImageView
+        android:id="@+id/project_image_view"
+        android:layout_width="@dimen/search_result_photo_width"
+        android:layout_height="@dimen/featured_search_result_photo_height"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentStart="true"
+        android:adjustViewBounds="true"
+        android:background="@drawable/gray_gradient"
+        android:scaleType="centerCrop"
+        tools:ignore="ContentDescription" />
+
+      <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        android:layout_below="@id/project_image_view"
+        android:layout_marginEnd="@dimen/grid_new_2"
+        android:layout_marginStart="@dimen/grid_new_2"
+        android:layout_marginTop="@dimen/grid_new_2"
+        android:orientation="vertical">
+
+        <TextView
+          android:id="@+id/project_name_text_view"
+          style="@style/SubheadPrimaryMedium"
+          android:textColor="@color/gray_black"
+          android:layout_width="fill_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginBottom="@dimen/grid_new_1"
+          android:maxLines="3"
+          tools:text="Name" />
+
+        <TextView
+          android:id="@+id/project_stats_text_view"
+          style="@style/ActivityFeedCard"
+          android:textColor="@color/black_alpha_50"
+          android:layout_width="fill_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginBottom="@dimen/grid_new_2"
+          android:maxLines="2"
+          android:text="@string/search_by_creator" />
+      </LinearLayout>
+
+    </RelativeLayout>
```

Comment:
can add a new line between these closing tags here, or consolidate them all together. we're not super finicky about xml style but it's nice to be consistent